sdk/computefleet/arm-computefleet/vitest.config.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { defineConfig } from "vitest/config";
```

Comment:
Would it be possible to use the base configs from the root instead of copy/pasting these files?  Unless there are some custonizations needed.

Reply:
@mpodwysocki are you suggesting something like the below? 
```
import { defineConfig, mergeConfig } from "vitest/config";
import { defineConfig as defaultConfig } from "../../vitest.shared.config.js";

export default mergeConfig(defaultConfig, defineConfig({}));
```